config.sh

```diff
@@ -53,7 +53,7 @@ config-cli () {
 
 login () {
 	echo "Setting api and login tools."
-	bx login --sso -a api.eu-gb.bluemix.net
+	bx login -a $LOGIN_URL
```

Comment:
One problem with the migration to resource groups is that the Org/Space targeting is no longer interactive.

Reply:
This can be resolved by adding `bx target --cf` below this line.